### PR TITLE
Adde location.html file with initial structure.

### PR DESCRIPTION
**Pull request by otarza**
Created location.html page.

Work in progress - TODO:
- [x] Create style;
- [x] todo 4
- [x] Create mobile screens
- [x] todo 3

### CODE REVIEW

File: location.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<html>
+  <head>
+  </head>
```
Comment (AlexTsamala) [inline location.html]: i dont like it change please

File: location.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  <body>
+    <h1> Location Page </h1>
```
Comment (AlexTsamala) [inline location.html]: i like it but change

**Review by AlexTsamala — CHANGES_REQUESTED** [review]

**Review by AlexTsamala — APPROVED** [review]
Looks great !!!

**Comment by lamodots** [comment]
@otarza What is the todo here?

**Comment by lamodots** [comment]
> Looks great !!!

What is the todo? what needs to be worked on?

**Pull request merged**